### PR TITLE
July commit of the month 

### PR DESCRIPTION
**Pull request by mariavivanco**
Hi @nicks @maiamcc, 

Please review the blog post I made for the month of July featuring the command line flag for no browser. Lmk your thoughts!

### CODE REVIEW

File: blog/_posts/2019-08-01-commit-of-the-month-july.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  - web-interface
+  - terminal
+  - tilt
+  - flag 
+---
+
+
+Welcome the second edition of Commit of the Month, a series of blog posts where we highlight some of the work done on Tilt this month! 
+
+This month’s Commit of the Month is the command line flag that disables the webUI:
```
Comment (maiamcc) [inline blog/_posts/2019-08-01-commit-of-the-month-july.md]: nit: it doesn't disable the web ui, just the auto-open when you run tilt.

File: blog/_posts/2019-08-01-commit-of-the-month-july.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+This one pops up on your _browser_..
+![Web UI](/assets/images/july-tilt-commit-of-the-month/webUI.png) 
+
+And this one pops up on your _terminal_.. 
+![Terminal UI](/assets/images/july-tilt-commit-of-the-month/TUI.png)
+
+Look familiar? 
+
+Well now, if you want to disable the web UI, you can now tilt up along with “--no-browser”
```
Comment (maiamcc) [inline blog/_posts/2019-08-01-commit-of-the-month-july.md]: same nit as before -- you're not disabling it, just making it not pop up. (i think the most common use of this is probably people who already have the web UI up in a tab and don't want tilt to pop up ANOTHER one)

File: blog/_posts/2019-08-01-commit-of-the-month-july.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+Welcome the second edition of Commit of the Month, a series of blog posts where we highlight some of the work done on Tilt this month! 
+
+This month’s Commit of the Month is the command line flag that disables the webUI:
+
+* [web: flag for opening browser](https://github.com/windmilleng/tilt/pull/1830)
+
+## What does it do?
+
+
+When you Tilt Up, usually you will see two UIs - the one that shows up on your terminal and the webUI.
```
Comment (maiamcc) [inline blog/_posts/2019-08-01-commit-of-the-month-july.md]: should probs be "web UI" (with a space) throughout

File: blog/_posts/2019-08-01-commit-of-the-month-july.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+Look familiar? 
+
+Well now, if you want to disable the web UI, you can now tilt up along with “--no-browser” 
+```
+tilt up --no-browser
+```
+
+## But.. why do we have two interfaces in the first place? 
+This is something we are still trying to figure out as well. At first, the terminal UI was the original Tilt - where you can see your services and their status. As we progressed through our Tilt journey, we experimented with building the web UI, which made it easier to navigate through logs and preview the services.
```
Comment (maiamcc) [inline blog/_posts/2019-08-01-commit-of-the-month-july.md]: maybe a note on why we bothered to auto-open it? (i.e. b/c we wanted to make sure ppl knew about it!)

File: blog/_posts/2019-08-01-commit-of-the-month-july.md (hunk 2 above)
Comment (mariavivanco) [reply blog/_posts/2019-08-01-commit-of-the-month-july.md]: Yes! Thank you this is a really good point

File: blog/_posts/2019-08-01-commit-of-the-month-july.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+
+On startup, Tilt automatically opens up a new tab on your browser displaying our web interface.  We made it this way because the web interface is fairly new and we want people to know about it.
+
+Well now, if you want to stop Tilt from automatically opening a new tab for the web UI, you can `tilt up` along with `--no-browser` like so:
+```
+tilt up --no-browser
+```
+
+## Wait.. shouldn't Tilt detect if a tab is already open?
```
Comment (maiamcc) [inline blog/_posts/2019-08-01-commit-of-the-month-july.md]:
```suggestion
## Wait... shouldn't Tilt detect if a tab is already open? 
```
(the ellipsis should always be three dots btw)

File: blog/_posts/2019-08-01-commit-of-the-month-july.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+## Wait.. shouldn't Tilt detect if a tab is already open? 
+It totally should!  
+
+Usually, Tilt is smart enough that it detects whether or not you already have Tilt browser tab open. 
+
+But then, one of our fellow Tilters came to us in the [*#Tilt* channel in the K8s slack](https://kubernetes.slack.com/messages/CESBL84MV/) with a use case we hadn’t thought about! 
+ 
+As they mentioned, if you keep the Tilt tab overnight, sometimes browsers like Chrome will decide that said tab isn’t active anymore and will put it to sleep. Since the tab is asleep, Tilt isn’t able to detect that the tab is already open, causing it to open another Tilt tab with the Tilt web UI. 
+
+However with our new handy dandy `--no-browser` flag. we can work around that and restrict Tilt from opening a new tab.
```
Comment (maiamcc) [inline blog/_posts/2019-08-01-commit-of-the-month-july.md]:
```suggestion
However with our new handy dandy `--no-browser` flag, we can work around that and restrict Tilt from opening a new tab.
```

File: blog/_posts/2019-08-01-commit-of-the-month-july.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+
+
+## But... why do we have two interfaces in the first place? 
+This is something we are still trying to figure out as well. The original Tilt was just the terminal UI-- your services and their status. As we progressed through our Tilt journey, we experimented with the web UI, which made it easier to navigate through logs and preview the services.
```
Comment (maiamcc) [inline blog/_posts/2019-08-01-commit-of-the-month-july.md]:
```suggestion
This is something we are still trying to figure out as well. The original Tilt was just the terminal UI--your services and their status, with logs for each service available on a keypress. As we progressed through our Tilt journey, we experimented with the web UI, which made it easier to navigate through logs and preview the services.  
```

Comment (maiamcc) [reply blog/_posts/2019-08-01-commit-of-the-month-july.md]: (TUI used to have log support, seemed good to mention)

File: blog/_posts/2019-08-01-commit-of-the-month-july.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+
+
+## But... why do we have two interfaces in the first place? 
+This is something we are still trying to figure out as well. The original Tilt was just the terminal UI-- your services and their status. As we progressed through our Tilt journey, we experimented with the web UI, which made it easier to navigate through logs and preview the services.  
+
+Love of people love the web UI, but then again there are advantages to having a terminal UI as well. For some engineers, it feels more similar to their day-to-day workflows. 
+
+Our next steps will be investigating what it would be like to show users just one UI or the other.
+
+Whether you prefer the terminal UI to the web UI, or you like the web UI so much that you keep it open for so long that Tilt thinks it’s not there anymore, we’ve got you covered.
```
Comment (maiamcc) [inline blog/_posts/2019-08-01-commit-of-the-month-july.md]: i still like this sentence but am not positive it belongs in this section -- maybe put in the section below, above all the release info?

File: blog/_posts/2019-08-01-commit-of-the-month-july.md [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+
+## But... why do we have two interfaces in the first place? 
+This is something we are still trying to figure out as well. The original Tilt was just the terminal UI-- your services and their status. As we progressed through our Tilt journey, we experimented with the web UI, which made it easier to navigate through logs and preview the services.  
+
+Love of people love the web UI, but then again there are advantages to having a terminal UI as well. For some engineers, it feels more similar to their day-to-day workflows. 
+
+Our next steps will be investigating what it would be like to show users just one UI or the other.
+
+
+## Let us know your experience using the flag!
```
Comment (nicks) [inline blog/_posts/2019-08-01-commit-of-the-month-july.md]: i would probably just delete this section. We already have a page footer for stuff like this

File: src/_data/people.yml [hunk 10]
```diff
@@ -20,6 +20,6 @@ han:
   github: yuindustries
 maria:
   name: Maria Vivanco
-  github: mariabelenvivanco
+  github: mariavivanco
```
Comment (nicks) [inline src/_data/people.yml]: :raised_hands:

File: blog/_posts/2019-08-01-commit-of-the-month-july.md (hunk 8 above)
Comment (maiamcc) [reply blog/_posts/2019-08-01-commit-of-the-month-july.md]: (not a huge deal tho -- like if you delete the last section, you can just leave this sentence here, or come up with another more wrap-up y section to put this in.)

**Pull request merged**